tests/common/fixtures/duthost_utils.py

```diff
@@ -38,13 +38,13 @@ def _backup_and_restore_config_db(duts, scope='function'):
 
     for duthost in duthosts:
         logger.info("Backup {} to {} on {}".format(CONFIG_DB, CONFIG_DB_BAK, duthost.hostname))
-        duthost.shell("cp {} {}".format(CONFIG_DB, CONFIG_DB_BAK))
+        duthost.fetch(src=CONFIG_DB, dest="/tmp/{}{}".format(duthost.hostname, CONFIG_DB_BAK), flat=True)
```

Comment:
It may be a good idea to still store the config db within the device, as the original idea.

The problem here was that /tmp/ was deleted across upgrades, and there was nothing to restore from.

If you agree, then you can backup the config in /host/ dire, instead of /tmp. Contents of /host/ will persist across reboots/upgrades.

Reply:
Sure, this works as well.